chaos/db_helper.py

```diff
@@ -190,31 +190,37 @@ def clean_message(msg, type=''):
         msg.text = msg.text.replace('\r\n', ' ')
 
 def manage_message(impact, json, client_id):
+    if 'messages' not in json:
```

Comment:
I'm not comfortbale with that, but perhaps a comment can do the trick
```suggestion
   #same management for every cases as 'messages' is not required
    if 'messages' not in json:
```